### PR TITLE
Added full license information to the project as required

### PR DESCRIPTION
**Pull request by patrickpl**
TODOS:
  * Add NOTICE file
  * Check / Add notice to every file
  * Check is CONTRIBUTORS file is needed

See also: http://www.apache.org/dev/apply-license.html#new

### CODE REVIEW

**Comment by patrickpl** [comment]
@martinsteger please let me know if this is fine for you and I will quickly do this for the other projects as well

**Comment by martinsteger** [comment]
@hannesgredler can you please answer Patric's query
@patrickpl by accepting this pull request,  the rtbrick copyright gets replaced by the Apache License copyright template. This needs to be fixed anyway.

**Comment by patrickpl** [comment]
The rtbrick copyright notice should be in the notice file

**Comment by patrickpl** [comment]
The rtbrick copyright notice should be in the notice file

**Comment by martinsteger** [comment]
Thanks for the quick response. I'm not an license expert and I appreciate all feedback to specify the license properly. I also consulted with Hannes and I'll update the license header in all source files and by doing so update the LICENSE file as well. It seems that the NOTICE file is not needed, when the copyright in the LICENSE file is replaced. I checked other open-source projects licensed under Apache 2. Docker, for example, replaced the copyright in the LICENSE file and has no additional NOTICE file.
Bottom line for now is to update the license header in all files and the LICENSE file without adding a NOTICE file.
@patrickpl can you do me a favor an get in touch with Hannes if you have concerns with this approach. I need Hannes confirmation to add the licensing information.

**Comment by patrickpl** [comment]
I hope I got you right, just for clarification:

1. The _LICENSE_ file **MUST** contain an unmodified version of the Apache License, Version 2.0 (SPDX: Apache-2.0)
2. A _notice in every source_ file **SHOULD** be a notice to clarify that this file has been released by RtBrick under the license in the license file.
3. A _NOTICE_ file **MAY** be created to clarify better the different source code owner of the different parts of the project. It helps conrtibutors to understand better the project. The NOTICE file gets more important if different copyright owners are included or in case different licenses are used e.g. CC for media files

@martinsteger I think you can reject the request, since you also want to add the changes for each file and this pull request only includes the license file change

@hannesgredler we may continue the discussion by mail if needed. Just let me know
